release-openyurt

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
you can exec `make release` command under openyurt repository to generate images of OpenYurt components.  it's like that this file is a local test script. and how about remove it from this PR?

Reply:
OK, I will remove it from this PR, it is kind of an enhancement, especially for local development, you can build and deliver, then load and deploy with one command